packages/proxychains-ng/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rofl0r/proxychains-ng
+TERMUX_PKG_DESCRIPTION="a preloader which hooks calls to sockets in dynamically linked programs and redirects it through one or more socks/http proxies. It is a kind of proxifier."
+TERMUX_PKG_VERSION=4.12
+TERMUX_PKG_MAINTAINER="Oliver Schmidhauser @Neo-Oli"
+TERMUX_PKG_SRCURL=https://github.com/rofl0r/proxychains-ng/releases/download/v${TERMUX_PKG_VERSION}/proxychains-ng-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=482a549935060417b629f32ddadd14f9c04df8249d9588f7f78a3303e3d03a4e
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS=" --prefix=${TERMUX_PREFIX}"
```

Comment:
this isn't needed, `build-package.sh` already inserts this in

Reply:
Right, thanks. I removed it in https://github.com/termux/termux-packages/pull/1575/commits/c59fed3a98d1d1d309453da0d5ad7acf513468b8